static/app/views/issueDetails/streamline/foldSection.tsx

```diff
@@ -175,7 +188,8 @@ export function FoldSection({
         id={sectionKey + additionalIdentifier}
         scrollMargin={navScrollMargin ?? 0}
         role="region"
-        aria-label={titleLabel}
+        // XXX: We should eventually only use titles as string, or explicitly pass them to stay accessible
+        aria-label={typeof title === 'string' ? title : sectionKey}
```

Comment:
This comment suggests we should just use [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby) here and point to the `TitleWrapper` element.